packages/extensions-sdk/src/cli/commands/add.ts

```diff
@@ -101,6 +101,11 @@ export default async function add(options: AddOptions): Promise<void> {
 			},
 		]);
 
+		if (extensionOptions.entries.map((entry) => entry.name).includes(name)) {
```

Comment:
Moving the map outside of the `if` condition should make it a bit clearer. We could use a `Set` here too.

```suggestion
		const exitingNames = new Set(extensionOptions.entries.map((entry) => entry.name));

		if (exitingNames.has(name)) {
```